app/bin/tools/backfill_package_fields.dart

```diff
@@ -60,6 +62,7 @@ Future<void> _backfillPackageFields(Package p) async {
       package.likes ??= 0;
       package.doNotAdvertise ??= false;
       package.isDiscontinued ??= false;
+      package.isUnlisted ??= package.isDiscontinued;
       package.assignedTags ??= [];
```

Comment:
Wouldn't it be best if we only made an insert, when we changed one of these properties?

Reply:
Do you mean setting the `Package.updated` property?

Reply:
No, I means, if we are changing no properties on the `package` entity, because there is nothing to backfill, we still call `tx.insert(package)`, isn't that slow?

Reply:
Oh, that is handled in lines 48-54. If all the potential changes have non-zero value, we just return before doing the transaction.